src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -106,43 +107,9 @@ export function transformVAOSAppointment(appt) {
 
   const start = timezone ? moment(appt.start).tz(timezone) : moment(appt.start);
 
-  return {
-    resourceType: 'Appointment',
-    id: appt.id,
-    status: appt.status,
-    // TODO Unclear what these reasons are
-    cancelationReason: appt.cancellationReason,
-    requestedPeriod: appt.requestedPeriods,
-    start,
-    // TODO: ask about created and other action dates like cancelled
-    created: null,
-    minutesDuration: appt.minutesDuration,
-    // TODO: ask about service types for CC and VA requests
-    type: {
-      coding: [
-        {
-          code: appt.serviceType,
-          display: getTypeOfCareById(appt.serviceType)?.name,
-        },
-      ],
-    },
-    reason: PURPOSE_TEXT.find(purpose => purpose.serviceName === appt.reason)
-      ?.short,
-    location: {
-      // TODO: what happens when vaos service can't find sta6aid for the appointment
-      vistaId: appt.locationId?.substr(0, 3),
-      clinicId: appt.clinic,
-      stationId: appt.locationId,
-    },
-    contact: {
-      telecom: appt.contact?.telecom.map(contact => ({
-        system: contact.type,
-        value: contact.value,
-      })),
-    },
-    preferredTimesForPhoneCall: appt.preferredTimesForPhoneCall,
-    comment: appt.comment,
-    videoData: {
+  let videoData = { isVideo };
```

Comment:
The changes in this transformer were to not return certain video or request fields when they aren't relevant. I did this to reduce the differences between the v0 and v2 transformed output. Ideally I think I would return all the request fields as null fields, but I'd have to do that in the two different v0 transformers and it seemed like too much extra work for now. But it might be better to do that instead.